jellyfin_kodi/entrypoint/default.py

```diff
@@ -529,7 +529,7 @@ def get_fanart(item_id, path, server_id=None, api_client=None):
         xbmcvfs.mkdirs(directory)
         item = api_client.get_item(item_id)
         obj = objects.map(item, 'Artwork')
-        backdrops = api.API(item).get_all_artwork(obj)
+        backdrops = api.API(item, None, api_client.config.data['auth.token']).get_all_artwork(obj)
```

Comment:
You don't have to add the implicit `None` here for the server.  It could be written as 
```suggestion
        backdrops = api.API(item, api_key=api_client.config.data['auth.token']).get_all_artwork(obj)
```

I'm not sure which method I like better, just pointing it out.